src/pages/gen2/build-a-backend/data/data-modeling/relationships/index.mdx

```diff
@@ -104,110 +114,164 @@ const schema = a.schema({
     tags: a.manyToMany('Tag', { relationName: 'PostTags'})
```

Comment:
Not part of this PR, but the Update "Has Many" example on L70 should use a different ID for the newly updated member. Perhaps the "Create" example can use `"ORIGINAL_MEMBER_ID"`, the "Update" example can use `"OTHER_MEMBER_ID"`, and the "Delete" example can use `"ORIGINAL_MEMBER_ID"`. That will reinforce the expectation that customers are responsible for maintaining the relationship integrity.